dynamodb-stream-eventbridge-fanout/src/main/java/no/unit/nva/doi/publisher/EventBridgePublisher.java

```diff
@@ -0,0 +1,107 @@
+package no.unit.nva.doi.publisher;
+
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent.DynamodbStreamRecord;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.time.Clock;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import no.unit.nva.doi.utils.JacocoGenerated;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequest;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequestEntry;
+
+public class EventBridgePublisher implements EventPublisher {
+
+    private static final ObjectMapper objectMapper = new ObjectMapper()
+        .setSerializationInclusion(JsonInclude.Include.NON_NULL);
+
+    public static final String EVENT_SOURCE = "aws-dynamodb-stream-eventbridge-fanout";
+    public static final String EVENT_DETAIL_TYPE = "dynamodb-stream-event";
+
+    private final EventBridgeRetryClient eventBridge;
+    private final EventPublisher failedEventPublisher;
+    private final String eventBusName;
+    private final Clock clock;
+
+    private static final Logger logger = LoggerFactory.getLogger(EventBridgePublisher.class);
+
+    @JacocoGenerated
+    public EventBridgePublisher(EventBridgeRetryClient eventBridge,
+                                EventPublisher failedEventPublisher,
+                                String eventBusName) {
+        this(eventBridge, failedEventPublisher, eventBusName, Clock.systemUTC());
+    }
+
+    /**
+     * Constructor for EventBridgePublisher.
+     *
+     * @param eventBridge   eventBridge
+     * @param failedEventPublisher  failedEventPublisher
+     * @param eventBusName  eventBusName
+     * @param clock clock
+     */
+    public EventBridgePublisher(EventBridgeRetryClient eventBridge,
+                                EventPublisher failedEventPublisher, String eventBusName, Clock clock) {
+        this.eventBridge = eventBridge;
+        this.failedEventPublisher = failedEventPublisher;
+        this.eventBusName = eventBusName;
+        this.clock = clock;
+    }
+
+    @Override
+    public void publish(final DynamodbEvent event) {
+        Instant time = Instant.now(clock);
+        List<PutEventsRequestEntry> requestEntries = event.getRecords()
```

Comment:
this method could contain the following steps:

1. `createPutEventEntries()`
2. `putEventsToEventBridge`
3. `putFailedEventEntriesToSqsQueue`